src/TokenAirdrop.hs

```diff
@@ -28,7 +29,8 @@ tokenAirdrop config = do
               )
               beneficiaries
 
-  mapM_ (\(_, bs) -> mapM_ print bs >> putStrLn "==============") txPairs
+  -- Save Our Consoles
+  -- mapM_ (\(_, bs) -> mapM_ print bs >> putStrLn "==============") txPairs
```

Comment:
This could be deleted or we could add a `verbose` flag to the config to print extra output.

Reply:
Oh I like a verbose flag, theres a few prints I could put under that